app/styles/_core.less

```diff
@@ -571,6 +571,32 @@ label.checkbox {
   }
 }
 
+.compute-resource {
+  .resource-size {
+    display: flex;
```

Comment:
Since we've switch to flexbox and changing these styles already, is it possible to make the input 100% wide? I think it would look much better in our forms since the other controls are 100%.

For instance, instead of:

![screen shot 2016-12-19 at 9 06 30 am](https://cloud.githubusercontent.com/assets/1167259/21315490/71207b20-c5ca-11e6-9b42-00fa0c9a01aa.png)

We could have this:

![screen shot 2016-12-19 at 9 06 07 am](https://cloud.githubusercontent.com/assets/1167259/21315484/64be9fb0-c5ca-11e6-9751-7d1dd5518676.png)